src/parsers/manifest/dash/common/indexes/template.ts

```diff
@@ -389,14 +388,26 @@ export default class TemplateRepresentationIndex implements IRepresentationIndex
     const { timescale } = this._index;
     const segmentTimeRounding = getSegmentTimeRoundingError(timescale);
     const scaledPeriodStart = this._periodStart * timescale;
+    const scaledRelativeStart = start * timescale - scaledPeriodStart;
```

Comment:
what do you think about adding parenthesis here to make it more explicit ?

Reply:
Done